pages/common/gdaladdo.md

```diff
@@ -0,0 +1,8 @@
+# gdaladdo
+
+> Build overview images of raster datasets.
+> More information: <https://gdal.org/programs/gdaladdo>.
+
+- Build overview images of a raster dataset using the "average" [r]esampling method:
+
+`gdaladdo -r {{average}} {{path/to/input.tif}}`
```

Comment:
If the example description is for `average` the token syntax shouldn't be applied in the example, because it's not a placeholder.
```suggestion
`gdaladdo -r average {{path/to/input.tif}}`
```